src/Colors/index.js

```diff
@@ -3,55 +3,23 @@ export const hexToRGBA = (hex, alpha = 1) => {
   return `rgba(${r},${g},${b},${alpha})`;
 };
 
-const primary = "#353F41";
-const primaryHover = "#05e6c8";
-const error = "#e94119";
-const errorBackground = "#fce6e4";
-const gray = "#cdcdcd";
-const darkgray = "#4c4c4c";
-const headerColor = "#575756";
-const hint = "#898988";
-const primarybackground = "#fff";
-const secondarybackground = "#f7f7f7";
-const cardbackground = "#fefefe";
-
-// maintaining these names colors to avoid introducing breaking changes (for now).
-export {
-  primary,
-  gray,
-  darkgray,
-  primarybackground,
-  secondarybackground,
-  error,
-  errorBackground,
-  primaryHover,
-  headerColor,
-  hint,
-  cardbackground
-};
-
-// this is v2
 const colors = {
-  brandPrimary: "#03D4BF",
-  brandAlternative: "#00BFAC",
-
+  primary: "#03D4BF",
+  destructive: "#E63644",
+  affirmative: "#189F59",
+  warning: "#D97508",
+  white: "#FFFFFF",
+  destructiveDim: "#FFE1E1",
+  affirmativeDim: "#E8F5E5",
+  warningDim: "#FFF6D1",
   textPrimary: "#353F41",
   textSecondary: "#7B7B7B",
   textInactive: "#BFBFBF",
+  textActive: "#00BFAC",
   divider: "#E0E0E0",
-
+  background: "#F8F7F5",
   backgroundPrimary: "#F5F5F5",
```

Comment:
background primary is not in our colour pallet?

Reply:
you request to remove backgroundPrimary and backgroundSecondary?

Reply:
Im asking what these are? if they arent in our colour pallet then why are they here and where did they get defined from? where are they used and what is the impact of not exporting these

Reply:
there is not in the palette and since thats a breaking change it can be removed. For some reason I had the feeling that this were being used but I was wrong. 
Will remove and request review.

Reply:
yes we should use this opportunity to completely align the colour exports with the design library, as we have breaking changes anyway